### DISCUSSION TITLE
Facing issues with line chart plot

### DISCUSSION THREAD

**Discussion by AymenJm09** [omnetpp/omnetpp#982] [Q&A] [answered]
Hi i'm using the LTe multicell model with voip traffic to simulate the performance of 4g lte.
i'm getting results, when i try to plot the sinr with line chart the snr over the time but starting from the max and no variation in it just like this photo 1. please i don't have the experience with omnet 6 to plot charts and i need to my final year project. i just wanna visualize snr and cqi over the time , or BELR i wanna meaning chart like photo 2 and 3.
how to do that maybe using compute or apply operations or modify something in the config chart i don't know really i need help guys.

![image](https://github.com/omnetpp/omnetpp/assets/134916015/1bcea216-a31b-46bb-9b54-399c2385cf6e)

![image](https://github.com/omnetpp/omnetpp/assets/134916015/fd24e301-503b-4aaa-90fe-631d119132fc)

![image](https://github.com/omnetpp/omnetpp/assets/134916015/60373349-5cf4-4b6c-be88-9aa0e96321f2)

**Comment by rhornig** [comment, marked as answer]
I'm not expert on charting or data analysis, but your chart title shows "Mean of SINR". i.e. it displays a time averaged value, which you should expect to converge to the MEAN of the value. My first guess: don't use time averaging...
